native/swift/Sources/wordpress-api/LoginAPI.swift

```diff
@@ -8,21 +8,84 @@ import WordPressAPIInternal
 import FoundationNetworking
 #endif
 
-public final class WordPressLoginClient {
+public final class WordPressLoginClient: Sendable {
 
     public protocol Authenticator {
         func authenticate(url: URL, callbackURL: URL) async -> Result<URL, Error>
     }
 
     private static let callbackURL = URL(string: "x-wordpress-app://login-callback")!
 
-    public enum Error: Swift.Error {
+    public enum Error: Swift.Error, Equatable {
```

Comment:
This `Equatable` conformance works for the test suite, but probably doesn't work _super_ well in production.

This shouldn't really matter because nobody should need that, but in a future PR I'd like to make this way more robust – the trick is that that a lot of the underlying types have different information, so comparing them isn't straightforward. I'd want to capture these from raw HTTP responses then write a whole bunch of tests. 

I don't think it's worth it at this point though, because nobody really needs that – it's just for the test suite.